2022 - LI - Solar Auxilia.cat

```diff
@@ -11859,6 +11901,27 @@ Limitations
             <cost name="Pts" typeId="d2ee-04cb-5f8a-2642" value="0"/>
           </costs>
         </selectionEntry>
+        <selectionEntry type="upgrade" import="true" name="Reconnaissance Pattern Cohorts" hidden="false" id="7e09-aae1-d5ff-d43e" publicationId="d882-d2a-5da1-92c4" page="179">
+          <rules>
+            <rule name="Reconnaissance Pattern Cohorts" id="7e04-8329-9f52-1bc1" hidden="false" publicationId="d882-d2a-5da1-92c4" page="179">
+              <description>Effects
+• A Detachment using this Cohort Doctrine may select Solar Auxilia  Hermes Light Sentinel Squadrons as Troops choices. Solar Auxilia Hermes Light Sentinel Squadrons selected as Compulsory Troops  choices gain the Line Unit Sub-type.
+• Solar Auxilia Veletaris Assault Squadrons in this Detachment gain the Outflank special rule.
+• Aethon Heavy Sentinel Squadrons may include up to 4 additional  Aethon Heavy Sentinels instead of up lo 2 additional Aethon Heavy Sentinels.
```

Comment:
```suggestion
• Aethon Heavy Sentinel Squadrons may include up to 4 additional Aethon Heavy Sentinels instead of up to 2 additional Aethon Heavy Sentinels.
```